samples/subsys/shell/bluetooth/server/prj.conf

```diff
@@ -0,0 +1,31 @@
+CONFIG_PRINTK=y
+CONFIG_SHELL=y
+CONFIG_LOG=y
+CONFIG_LOG_CMDS=y
+CONFIG_INIT_STACKS=y
+CONFIG_THREAD_STACK_INFO=y
+CONFIG_KERNEL_SHELL=y
+CONFIG_THREAD_MONITOR=y
+CONFIG_BOOT_BANNER=n
+CONFIG_THREAD_NAME=y
+CONFIG_DEVICE_SHELL=y
+CONFIG_POSIX_CLOCK=y
+CONFIG_DATE_SHELL=y
+CONFIG_THREAD_RUNTIME_STATS=y
+CONFIG_THREAD_RUNTIME_STATS_USE_TIMING_FUNCTIONS=y
+CONFIG_STATS=y
+CONFIG_STATS_SHELL=y
+CONFIG_SHELL_LOG_BACKEND=n
+
+CONFIG_BT=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_SHELL_BACKEND_BT=y
+CONFIG_BT_DEVICE_NAME="Shell-Sample"
+CONFIG_BT_RX_STACK_SIZE=2048
+# Allow for large Bluetooth data packets.
+CONFIG_BT_L2CAP_TX_MTU=512
+CONFIG_BT_BUF_ACL_RX_SIZE=502
+CONFIG_BT_BUF_ACL_TX_SIZE=502
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
+
+CONFIG_ASSERT=y
```

Comment:
this feels like this could use some cleanup? For the sake of keeping the sample as "minimal" as possible, please make sure there isn't more in there than what is required for demonstrating the functionality.

Reply:
I'll look into it!

Reply:
I re-organized the prj.conf to make it more readable. Most of those config's are just for the sake of having commands to demo. I can further simplify it if it need be.